src/test/java/org/opendevstack/provision/services/MailAdapterTest.java

```diff
@@ -14,46 +14,41 @@
 
 package org.opendevstack.provision.services;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 
 import javax.mail.internet.MimeMessage;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.mockito.MockitoAnnotations;
 import org.opendevstack.provision.authentication.crowd.CrowdAuthenticationManager;
 import org.opendevstack.provision.model.OpenProjectData;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
-@DirtiesContext
 @ActiveProfiles("crowd")
 public class MailAdapterTest {
-  @Mock JavaMailSender mailSender;
 
-  @InjectMocks @Autowired MailAdapter mailAdapter;
+  @Autowired private MailAdapter mailAdapter;
 
-  @Autowired CrowdAuthenticationManager manager;
+  @Mock private CrowdAuthenticationManager crowdAuthenticationManager;
+
+  @Mock private JavaMailSender mailSender;
```

Comment:
same as last comment

Reply:
Actually, that one has to stay as Mock, otherwise you will get 'Bean must not be empty' exception and it has been Mock already before me changing anything.

Reply:
I see... thanks!